package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@mechanicalrock/cdk-constructs",
+  "version": "0.0.2",
+  "description": "CDK constructs to enable creation of AWS resources for projects.",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "test": "jest --verbose --coverage",
+    "build": "rm -rf dist && tsc"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/MechanicalRock/cdk-constructs.git"
+  },
+  "keywords": [
+    "cdk",
+    "constructs"
+  ],
+  "author": "mechanical rock",
+  "license": "GPL-3.0",
```

Comment:
Any particular reason for the GPL licence?

You probably want to Apache-2.0, in keeping with the CDK license, of which this is derivative of.

Reply:
I wans't sure about this. I will update.